src/main/java/com/gmail/yuramitryahin/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.gmail.yuramitryahin.service;
+
+import com.gmail.yuramitryahin.dao.ManufacturerDao;
+import com.gmail.yuramitryahin.lib.Inject;
+import com.gmail.yuramitryahin.lib.Service;
+import com.gmail.yuramitryahin.modal.Manufacturer;
+import java.util.List;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return manufacturerDao.get(id)
+                .orElseThrow(() -> new RuntimeException("Manufacturer not found!"));
```

Comment:
this message is not very informative.
let's change it or use method `get()` instead of `orElseThrow()`

Reply:
fixed